c/ReachSafety-Combinations.set

```diff
@@ -0,0 +1 @@
+combinations/*.yml
```

Comment:
`.set` files should now contain a short comment with description.

Reply:
😓 
So the `.cfg` file is gone, instead the description moved to the README in the corresponding subdirectories,
and at some point it was decided that we also have short descriptions in the .set files.

I guess it would be easier for both contributor and reviewer to have these requirements visible at some prominent location, for example the CONTRIBUTING.md - currently it's trial & error at multiple locations for me. (sorry if I just didn't find the info in the documentation)

Reply:
Hm, it's true that the description in `.set` files is not documented right now. But there were actually less changes, it is just that the description from `.cfg` files was moved to `.set` files. Nothing changed for readmes.

Reply:
Ah ok. In one of the last reviews you provided, you noted that I can remove the `.cfg` file because its information moved to the README. But then that corresponding .set-file was probably already adjusted by you, and that's why I didn't notice that change.

Reply:
Weird, I can remember the comment you are referring to, but I can't find it anymore. I think I mentioned that the architecture was moved to task-definition files, and if I wrote any other incorrect information, I am sorry about that.

So the definitive info is: `Architecture` of `.cfg` files is now `data_model` in task-definition files (with different syntax), `Description` of `.cfg` files should now be a comment in the respective `.set` file (and of course a longer description should be in the readme, like before).